docs/README.md

```diff
@@ -7,6 +7,7 @@ Table of contents
 - [Bills](io.cozy.bills.md): bills
 - [Contacts](io.cozy.contacts.md): instance owner contacts
 - [Files](io.cozy.files.md): files documents
+- [Konnectors](io.cozy.konnectors): Connectors
 - [Notifications](io.cozy.notifications.md): notifications made by the apps
 - [Photos Albums](io.cozy.photos.albums.md): photos albums
 - [Remote requests](io.cozy.remote.requests.md): logs of requests via the remote doctypes
```

Comment:
After merging the two PRs, can you add the two links to the `toc.yml` so that it appears in the menu of https://docs.cozy.io ?